doc/GRPC_DAG_SERVICE.md

```diff
@@ -0,0 +1,9 @@
+# gRPC Based DAGService
+
+One of the core interfaces for using IPFS is the [`DAGService`](https://github.com/ipfs/go-ipld-format/blob/master/merkledag.go#L54) interface, that specifies how you can get, and add IPLD nodes both to the network, and your local blockstore.
+
+Within the `go` folder in the repository root is a file `dag_service.go` which satisfies the `DAGService` interface, allowing you to integrate TemporalX into a variety of other projects such as go-ipfs, ipfs-lite, and basically any other golang based project that uses the `DAGService` interface.
```

Comment:
I don't think where the code is is important. More so that the pb.DAGService satisfies ipld.DAGService using a gRPC client of TemporalX/

Reply:
good idea, instead of mentioning where it is I replaced that with a relative link and instead focused more on explaining the module.